otter/integration/lib/test_nova.py

```diff
@@ -5,14 +5,45 @@
 from twisted.trial.unittest import SynchronousTestCase
 
 from otter.integration.lib.nova import NovaServer
+from otter.util.http import headers
 
 
-class _Response(object):
+class Response(object):
     """Fake response object"""
     def __init__(self, code):
         self.code = code
 
 
+def get_fake_treq(test_case, method, url, expected_args_and_kwargs, response):
```

Comment:
Non-blocking: Maybe this or `StubTreq` should goto `treq` package to avoid test duplication?